### PR TITLE
fix #61662 ettoday.net

### LINKED ISSUE #61662: ettoday.net
### Issue URL (Ads)
 [https://www.ettoday.net/news/focus/%E5%9C%8B%E9%9A%9B/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.ettoday.net%2Fnews%2Ffocus%2F%25E5%259C%258B%25E9%259A%259B%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.12
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br/>Send Do-Not-Track header, <br/>Remove X-Client-Data header from HTTP requests, <br/>Strip URLs from tracking parameters, <br/>Self-destructing third-party cookies (2880), <br/>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances

### PR DESCRIPTION
**Pull request by stanislav-atr**
#61662

### CODE REVIEW

File: AnnoyancesFilter/sections/annoyances.txt [hunk 1]
```diff
@@ -1679,6 +1679,7 @@ gizmodo.com,kotaku.com###newsletterPromoModule
 cnet.com,newyorker.com##.newsletter-subscribe-form
 newyorker.com#?#div[class^="PageSections__container"] > section[class^="SoloPromoSection__promoWrapper"]:has(> div[class^="SoloPromoSection__promoColumn"] > p[class^="SoloPromoSection__promo"] > span > a[href="https://www.newyorker.com/newsletters"])
 belretail.by##.bannerAfterArticle
+ettoday.net#?#.inner > div.block:has(.sidebar-hot-express > div > .piece > a[target="_blank"] > img)
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/annoyances.txt:1682]: It seems `> div > .piece > a[target="_blank"] > img` is redundant.

File: AnnoyancesFilter/sections/annoyances.txt [hunk 2]
```diff
@@ -1679,6 +1679,7 @@ gizmodo.com,kotaku.com###newsletterPromoModule
 cnet.com,newyorker.com##.newsletter-subscribe-form
 newyorker.com#?#div[class^="PageSections__container"] > section[class^="SoloPromoSection__promoWrapper"]:has(> div[class^="SoloPromoSection__promoColumn"] > p[class^="SoloPromoSection__promo"] > span > a[href="https://www.newyorker.com/newsletters"])
 belretail.by##.bannerAfterArticle
+ettoday.net#?#.inner > div.block:has(> .sidebar-hot-express)
```
Comment (zloyden) [inline AnnoyancesFilter/sections/annoyances.txt:1682]: Don't forget about tags after `>`

**Review by zloyden — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/annoyances.txt (hunk 2 above)
Comment (stanislav-atr) [reply AnnoyancesFilter/sections/annoyances.txt:1682]: You mean as in `div.sidebar-hot-express`?

Comment (zloyden) [reply AnnoyancesFilter/sections/annoyances.txt:1682]: Yes

Comment (stanislav-atr) [reply AnnoyancesFilter/sections/annoyances.txt:1682]: oh ok
though we dont need em  in for a short selectors like this one

**Review by zloyden — APPROVED** [review]

**Pull request merged**